src/PublicShareSidebarTrigger.vue

```diff
@@ -0,0 +1,69 @@
+<!--
+  - @copyright Copyright (c) 2022 Daniel Calviño Sánchez <danxuliu@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<Button @click="$emit('click')">
+		<MenuPeople :size="20"
+			fill-color="var(--color-primary-text)"
+			decorative />
+	</Button>
+</template>
+
+<script>
+import Button from '@nextcloud/vue/dist/Components/Button'
+import MenuPeople from './components/missingMaterialDesignIcons/MenuPeople.vue'
+
+export default {
+
+	name: 'PublicShareSidebarTrigger',
+
+	components: {
+		Button,
+		MenuPeople,
+	},
+
+}
+</script>
+
+<style lang="scss" scoped>
+/* Match style for header buttons defined in server stylesheets. */
+.button-vue {
```

Comment:
Sound like the button component needs another mode,
Can you raise an issue please in the vue repo?

Reply:
I will do tomorrow.

Reply:
Reported at https://github.com/nextcloud/nextcloud-vue/issues/2869 so it's not lost.

Reply:
> I should not have used a button component but a div with role="button" and class="menutoggle", as that is what is used in all header buttons.

That is what you do right now, correct? Because it does not work with keyboard navigation e.g. so still an issue.
But I'm fine proceeding as at least the button is back and clickable with the mouse for now.